src/main.js

```diff
@@ -100,6 +100,29 @@ app.on('ready', () => {
         }
     ];
 
+    /* On macOS, cut/copy/paste doesn't work by default unless the options
+    are added to the edit menu, or some custom clipboard API implementation is
+    used. On Windows and Linux, these work out of the box...so only add on macOS */
+    if (process.platform === 'darwin') {
+        menuTemplate[1].submenu.push(
+            {
+                type: 'separator'
+            }, {
+                label: 'Cut',
+                accelerator: 'CmdOrCtrl+X',
```

Comment:
In hindsight, I guess these could be changed to just `Cmd+` ... but hey, they don't do any harm as they are, and maybe in future we may want to implement something custom cross-platform